platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -30,6 +30,28 @@ spec:
         - sidecar
         - -v
         - "2"
+        - --configPath
```

Comment:
What happened with the default values ? We are past freeze date, plenty of engineers testing and having different configs ( including the deb for raw VM ) - there is no valid reason for requiring any of those params.

This is not the first program to not be used primarily by end users - the common practice is to have sensible defaults.

Reply:
It's the result of admission controller processing the flags. We can't be sure agent and discovery on defaults, so it's better to be explicit.